lib/ui/painting.dart

```diff
@@ -1708,7 +1718,7 @@ void decodeImageFromPixels(
 ) {
```

Comment:
why not support targetWidth and targetHeight here too?

Reply:
Done